docs/_posts/0000-03-01-object-orientation.md

```diff
@@ -225,19 +225,14 @@ Note: Different because most languages only have one way to do this
 
 ### A simplified class diagram
 
-```text
-                    ┌──────────────┐
-                    │  Class Name  │
-                    │              │
-                    │  Attributes  │
-                    │              │
-                    │  Methods     │
-                    └──────▲───────┘
-                           │
-                           │inheritance
-                     ┌─────┴──────┐
-                     │Child Class │
-                     │            │
-                     └────────────┘
-```
-
+<mermaid>
+classDiagram
+    Parent Class <|-- Child Class 
+    class Parent Class {
+        attribute1
+        attribute2
+        method1()
+        method2()
+    }
+    class Child Class
```

Comment:
Trying to include all the ways we use arrows - probably needs expanding:
```suggestion
classDiagram
    Parent Class <|-- Child Class : implements / extends
    class Parent Class {
        attribute1
        attribute2
        method1()
        method2()
    }
    ParentClass --> SomeOtherClass : callSomeMethod()
    class Child Class
```

Reply:
Yep, good idea. Labelling is sensible.